kie-spring-boot/pom.xml

```diff
@@ -33,7 +33,7 @@
   </modules>
 
   <properties>
-    <version.me.snowdrop.narayana>2.4.1</version.me.snowdrop.narayana>
+    <version.me.snowdrop.narayana>2.6.1.redhat-00001</version.me.snowdrop.narayana>
```

Comment:
don't use redhat specific versions here please. just community ones without redhat-#### suffix

Reply:
@mareknovotny this comes from the fact that dependencies should be aligned with ROAR, which declares that one:
https://download.eng.brq.redhat.com/released/JBoss-middleware/rhoar/spring-boot/2.4.9/extras/repository-artifact-list.txt

Is it the same without redhat-#### suffix?

Reply:
yes, that productized version is orchestrated automatically in build if it exists

Reply:
Sure, will update accordingly.